crates/iceberg-catalog/migrations/20240913122911_users.sql

```diff
@@ -0,0 +1,14 @@
+create table users
```

Comment:
one table with a bunch of nullable columns or one for natural persons and another for service users?

Reply:
iirc, name is optional in tokens, should we reject tokens without name?